pytest-embedded-idf/pytest_embedded_idf/dut.py

```diff
@@ -76,11 +91,73 @@ def toolchain_prefix(self) -> str:
         else:
             raise ValueError(f'Unknown target: {self.target}')
 
+    @property
+    def panic_output_decode_script(self) -> Optional[str]:
+        """
+        Returns:
+            Panic output decode script path
+        """
+        script_filepath = self._panic_output_decode_script or os.path.join(
+            os.getenv('IDF_PATH', 'IDF_PATH'),
+            'tools',
+            'gdb_panic_server.py',
+        )
+        if not os.path.isfile(script_filepath):
+            logging.info(
+                'Panic output decode script not found. Please use --panic-output-decode-script flag '
+                'to provide script or set IDF_PATH (Default: $IDF_PATH/tools/gdb_panic_server.py)'
+            )
+            raise ValueError
```

Comment:
raise ValueError('...') with your logging message above would be better.

The log message won't be printed if `--log-cli-level` is unset.